pages/common/autossh.md

```diff
@@ -1,29 +1,29 @@
 # autossh
 
 > Runs, monitors and restarts SSH connections.
-> Auto-reconnects to keep port forwarding tunnels up. Accepts all ssh flags.
+> Auto-reconnects to keep port forwarding tunnels up. Accepts all `ssh` flags.
 > More information: <https://harding.motd.ca/autossh>.
 
-- Open an SSH session, restarting when a monitoring port fails return data:
+- Start an SSH session, restarting when a monitoring port fails return data:
 
-`autossh -M {{monitor_port}} {{ssh_command}}`
+`autossh -M {{monitor_port}} "{{ssh_command}}"`
 
-- Open an SSH session which forwards a local port to a remote one, restarting if necessary:
+- Forward a local port to a remote one, restarting when necessary:
 
 `autossh -M {{monitor_port}} -L {{local_port}}:localhost:{{remote_port}} {{user}}@{{host}}`
 
-- Fork before executing ssh (runs in the background) and don't open a remote shell:
+- Fork `autossh` into the background before executing `ssh` and don't open a remote shell:
 
-`autossh -f -M {{monitor_port}} -N {{ssh_command}}`
+`autossh -f -M {{monitor_port}} -N "{{ssh_command}}"`
 
-- Run autossh in the background, with no monitoring port, instead relying on SSH keep-alives every 10 seconds to detect failure:
+- Run in the background, with no monitoring port, instead sending SSH keep-alive packets every 10 seconds to detect failure:
```

Comment:
```suggestion
- Run in the background, with no monitoring port, instead send SSH keep-alive packets every 10 seconds to detect failure:
```

Reply:
Good spot, I'd say `..., and instead send ...`.  Can you update the suggestion?

Reply:
Updated

Reply:
Ping @bl-ue